zio-http/jvm/src/main/scala/zio/http/netty/server/ServerInboundHandler.scala

```diff
@@ -80,7 +80,10 @@ private[zio] final case class ServerInboundHandler(
         try {
           if (jReq.decoderResult().isFailure) {
             val throwable = jReq.decoderResult().cause()
-            attemptFastWrite(ctx, Response.fromThrowable(throwable))
+            attemptFastWrite(
+              ctx,
+              Response.fromThrowable(throwable, runtime.unsafeRunSync(ErrorResponseConfig.configRef.get)),
```

Comment:
It seems a bit wasteful to start a fiber only to extract a `FiberRef` from it.

How about adding this method to `NettyRuntime`:

```scala
def getRef[A](ref: FiberRef[A]): A = zioRuntime.fiberRefs.getOrDefault(ref)
```

And then this becomes:
```scala
Response.fromThrowable(throwable, runtime.getRef(ErrorResponseConfig.configRef))
```